SMARTAGRIFOOD/wineTraceability/README.md

```diff
@@ -0,0 +1 @@
+Readme for the data models of the Wine Traceability Data project
```

Comment:
This file is not necessary (it is overwritten when the model is officially publish)